packages/mobile/src/send/SendAmount.tsx

```diff
@@ -202,9 +209,26 @@ function SendAmount(props: Props) {
     }
   }, [props.route, localCurrencyCode, localCurrencyExchangeRate, dollarAmount])
 
+  const onReviewButtonPressed = async () => {
+    setReviewButtonToPressed(true)
+    // Wait until verification status is known to proceed
+    while (recipientVerificationStatus === RecipientVerificationStatus.UNKNOWN) {
```

Comment:
Are there any cases where the RecipientVerificationStatus stays UNKOWN? If so could add a timeout?

Reply:
I don't think that should ever happen. It is only `UNKNOWN` when the `fetchAddressesAndValidateSaga` is running. When that saga completes or fails, then the value will resolve to either `VERIFIED` or `UNVERIFIED`.